cmd/hz/internal/protobuf/tags.go

```diff
@@ -258,7 +258,7 @@ func m2s(mt model.Tag) (ret [2]string) {
 
 func reflectJsonTag(f protoreflect.FieldDescriptor) (ret model.Tag) {
 	ret.Key = "json"
-	ret.Value = checkSnakeName(f.JSONName())
+	ret.Value = checkSnakeName(f.Name())
```

Comment:
I think you need to fill in the pr title and description according to the pull request template.

Reply:
done